osu.Game/Overlays/Settings/Sections/Graphics/RendererSettings.cs

```diff
@@ -67,11 +67,7 @@ private void load(FrameworkConfigManager config, OsuConfigManager osuConfig, IDi
                 if (r.NewValue == RendererType.Automatic && automaticRendererInUse)
                     return;
 
-                if (game?.RestartAppWhenExited() == true)
-                {
-                    game.AttemptExit();
-                }
-                else
+                if (game?.Restart() != true)
```

Comment:
If I thinking this right, on Windows, changing the renderer will just outright restart. Would still expect a dialog here, but different wording that it will restart automatically.

Reply:
Don't really think its necessary.